kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -343,9 +341,18 @@ window.TopicContainerInnerView = BaseView.extend({
         this.trigger('back_button_clicked', this.model);
     },
 
-    node_by_slug: function(slug) {
+    deferred_node_by_slug: function(slug, callback) {
         // Convenience method to return a node by a passed in slug
```

Comment:
This doesn't actually return anything now, right? And neither does `node_by_slug`.